examples/rust/poseidon2/Readme.md

```diff
@@ -0,0 +1,5 @@
+# Using Poseidon2 API
+
+Sanity checks with standard sage output from attached sage script. The sage script originally from [Horizen labs code](https://github.com/HorizenLabs/poseidon2/blob/main/poseidon2_rust_params.sage) has been modified to print in standard format.
+
+Note that the digest element of the Poseidon2 hash api is the output state of Poseidon2 at index 1.
```

Comment:
Do we want this note to be more visible? Maybe an admonition would be better

Reply:
As far as I understand it everyone that uses Poseidon2 uses index 1.
Karthik - right? If it's the case then I don't see a reason to make it more visible.

Reply:
Yes generally so, but it is a convention and it is ok to state it.